### ISSUE TITLE
some action doesn't include AttackType

### ISSUE THREAD

**Issue by LittleNightmare** [lmcintyre/DamageInfoPlugin#5]
like action `22803` Tender Loin doesn't include AttackType. However, it has been used in ColorCastBar. The name of action '22804' is the same as action `22803` but it includes AttackType. This problem causes the type of action `22803` to become a `DamageType.Unknown`.  

i think the program could use ActionCategory instead of AttackType when not available.

**Comment by lmcintyre** [comment]
Hi, I don't think ActionCategory is reliable for determining if an action is magic damage or not. I can try to find another solution to these kinds of issues

**Comment by LittleNightmare** [comment]
how about search another action with the same name, like below
```c#
                if (tmpType == DamageType.Unknown)
                {
                    foreach (var tmp in actionSheet)
                    {
                        if ((DamageType)tmp.AttackType.Row == DamageType.Unknown || row.Name == null || row.Name == "") continue;
                        if (tmp.Name == row.Name) tmpType = (DamageType)tmp.AttackType.Row;
                    }
```

**Comment by lmcintyre** [comment]
There is no guarantee that two actions with the same name will be the same damage type

**Comment by LittleNightmare** [comment]
yeah, you are right. I find some actions like Terminus Est, Fireball with the same name but different AttckType. but most of them, are using the same type. Maybe could use current finding write a filter? Set it as an option of plugin. In addition, it causes more time when initializing the plugin (close to crash the game lol).
